src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -455,7 +455,7 @@ private void AddNameAndTypeArgumentsOrParameters(INamedTypeSymbol symbol)
 
             return;
 
-            void addExtensionParameter(NamedTypeSymbol? underlyingTypeSymbol) // PROTOTYPE use public API once it's available
```

Comment:
> // PROTOTYPE use public API once it's available [](http://example.com/codeflow?start=77&length=48)

It doesn't look like this comment is adressed

Reply:
The calling method has this internal symbol so I don't think this comment is applicable (even though we now have a public API for `ExtensionParameter`)

Reply:
> The calling method has this internal symbol so I don't think this comment is applicable (even though we now have a public API for ExtensionParameter)

Presence of the internal symbol is not the primary concern. Symbol display must be implemented in terms of public API, unless the API is not available. There is no guarantee that symbols that we asked to display are C# symbols. In addition, the code should not be written under assumption that ```underlyingTypeSymbol``` is not null, even when the input claims it is an extension